src/api.ts

```diff
@@ -54,9 +55,38 @@ router.post('/ai/summary/:id', async (req, res) => {
     }
 
     return rpcSuccess(res.status(200), summary, id);
-  } catch (e) {
+  } catch (e: any) {
     capture(e);
-    return rpcError(res, 'INTERNAL_ERROR', id);
+    return rpcError(res, e.message || 'INTERNAL_ERROR', id);
+  }
+});
+
+router.post('/ai/tts/:id', async (req, res) => {
```

Comment:
Any reason to use `post` here instead of `get` and for summery when we are not using any input from `body`

Reply:
Very basic bot protection, to avoid link prefetch or any other script/tools calling this API endpoint